### DISCUSSION TITLE
part-ii/2020-123/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#125] [Question comments]
Part II 2020 Probability and Measure: Paper 4, Section II, K 

https://questions.tripos.org/part-ii/2020-123/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a) $\mathbb{E}\left[S_n^4\right]$
(b) (i) completeness of $L^2$. $L^2$ convergence implies $L^1$ convergence implies convergence in measure implies weak convergence = convergence in law
(ii) 
(iii) characteristic function and Levy's criterion.

</details>
